bindings/python/parsers/python/model.cpp

```diff
@@ -28,11 +26,6 @@ namespace pinocchio
       // Get a dict for the global namespace to exec further python code with
       bp::dict globals = bp::extract<bp::dict>(main_module.attr("__dict__"));
 
-      // We need to link to the pinocchio PyWrap. We delegate the dynamic loading to the python
-      // interpreter.
-      bp::object cpp_module(
-        (bp::handle<>(bp::borrowed(PyImport_AddModule("libpinocchio_pywrap")))));
```

Comment:
@jcarpent  I removed this part and I don't see the point to use it. Tests are working fine on Linux and OSX (Windows is still running)

Reply:
Fine